biztalk/core/operational-data-service.md

```diff
@@ -0,0 +1,73 @@
+> [PowerBI.com](https://powerbi.microsoft.com) is a great resource to understand and learn more about Power BI.
+
+## Prerequisites
+* Download and install [Power BI Desktop](https://powerbi.microsoft.com/desktop/) on any computer that has network access to your [!INCLUDE[btsBizTalkServerNoVersion_md](../includes/btsbiztalkservernoversion-md.md)]
+* Install IIS on the [!INCLUDE[btsBizTalkServerNoVersion_md](../includes/btsbiztalkservernoversion-md.md)]. 
+* [Configure the REST APIs](../install-and-config-guides/configure-biztalk-server.md) for **Operational data service**.
+* Optional. Install and configure a [Power BI Gateway](https://powerbi.microsoft.com/gateway/) to connect [PowerBI.com](https://powerbi.microsoft.com) with your on-premises BizTalk Server. If you're not using an on-premises BizTalk Server, then you don't need the gateway.
+
+
+## Use Power BI to Load Operational Data.
+
+1. Download and install the [Power BI Desktop](https://powerbi.microsoft.com/desktop/) on your BizTalk Server. You can select to open it, which is optional. If you have a work or school account, you may have access to Power BI. Try signing in with that account. Or, you can try it for free after signing up.
+2. Open the `\Program Files (x86)\Microsoft BizTalk Server 2016\OperationalDataService` folder, and open the `BizTalkOperationalData.pbit` file:
+![Open pbit file](../core/media/operational-data-pbit.png)
+
+3. Power BI desktop opens, and you are prompted for a URL. Enter the `http://localhost/<yourWebSite>` URL that you created for your OData feed. For example, enter `http://localhost/BizTalkOperationalDataService`. Your URL looks similar to the following:
+![Enter the URL](../core/media/operational-data-url.png)
+
+4. Select **Load**. The window loads and connects to the different oData sources in the BizTalkOperationalDataService.json file. When it completes, the dashboard shows details about your environment.
+
+## Couldn't authenticate
+If you get `couldn't authenticate with the credentials provided` message similar to the following, then it’s possible your application pool identity doesn’t have enough access to the BizTalk Server databases. You can change the appPool identity within IIS to an account with more privileges, maybe your signed-in user account (which has local admin privileges).
+
+![Couldn't authenticate with the credentials provided](../core/media/operational-data-authentication-error.png)
+
+## Do more
+This is just the beginning. Power BI also has a gateway that can be installed on an on-premises BizTalk Server. Using the gateway, you can publish your dashboard, get real-time data, and create a schedule to refresh the dashboard. The following blog does a great job detailing these steps:
+
+* [How to publish BizTalk operational data on Power BI – Step-by-step configuration](https://blog.sandro-pereira.com/2017/05/07/biztalk-server-2016-feature-pack-1-how-to-publish-biztalk-operational-data-power-bi-step-by-step-configuration-part-3/)
```

Comment:
this link is not for an official documentation - should it be included here?

Reply:
This was there, and i don't find any real reason to remove it, let's keep it.

Reply:
We can make a general call on this topic - referencing external content within official Microsoft documents.
For restructuring. I am staying focused on the restructure aspect for the time being